src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,50 @@
+package duke.task;
+
+public abstract class Task {
+    public static final String OPEN_BRACKET = "[";
+    public static final String CLOSE_BRACKET = "]";
+    public static final String SPACE = " ";
+
+    protected String description;
+    protected boolean isDone;
+    protected String typeIcon;
+    protected String date;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); //mark done task with X
```

Comment:
Consider writing comments above the code, indented relative to the position of the code, as per coding standards.